core/controllers/reader.py

```diff
@@ -1794,3 +1794,21 @@ def post(self, exploration_id):
             )
 
         self.render_json(self.values)
+
+
+class CheckpointsFeatureStatusHandler(base.BaseHandler):
+    """The handler for checking whether the checkpoints feature is enabled."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {}
+    }
+
+    @acl_decorators.open_access
+    def get(self):
+        self.render_json({
+            'checkpoints_feature_is_enabled': (
+                config_domain.CHECKPOINTS_FEATURE_IS_ENABLED.value)
+        })
```

Comment:
I think the plan is actually to use config_domain and the admin page for this, since it's more well-tested. Let's start with that for now since this needs to go out soonish.

Reply:
Sounds good, if we plan to do it at a later time, please add a TODO comment + issue. Thanks!

Reply:
Hi Kevin! I have made the issue #15682 regarding the same. Also added a TODO in config_domain.